Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PowderILLEfficiency.py

```diff
@@ -344,7 +356,7 @@ def _configure_sequential(self, raw_ws):
             Configures the calibration with SequentialSummedReference1D method (D20)
             @param : the name of the raw detector scan (merged) workspace
         """
-        self._scan_points = mtd[raw_ws].getRun().getLogData('ScanSteps').value
+        self._scan_points = int(mtd[raw_ws].getRun().getLogData('ScanSteps').value)
```

Comment:
Is this cast necessary? It seems that the value type is always an integer.

Reply:
Yes, this is a calculated log, so might be float depending on the scan settings.